src/widgets/biometrics/biometrics-overview.component.tsx

```diff
@@ -116,8 +116,30 @@ const BiometricsOverview: React.FC<BiometricsOverviewProps> = ({ config }) => {
     if (tableRows.length) {
       return (
         <div className={styles.biometricsWidgetContainer}>
-          <div className={styles.biometricHeaderContainer}>
-            <h4>Biometrics</h4>
+          <div className={styles.biometricsHeaderContainer}>
+            <h4 className={`${styles.productiveHeading03} ${styles.text02}`}>
+              {headerTitle}
+            </h4>
+            <div className={styles.toggleButtons}>
+              <Button
+                className={styles.toggle}
+                size="field"
+                hasIconOnly
+                kind={chartView ? "ghost" : "secondary"}
+                renderIcon={Table16}
+                iconDescription="Table View"
```

Comment:
Are these descriptions displayed? If yes, we should translate them.

(Comment applies to all occurrences)

Reply:
Yes, they're displayed on hover. Translations are incoming!